app/view_models/prior_authority/v1/additional_cost.rb

```diff
@@ -33,7 +33,7 @@ def total_item_cost(original)
       end
 
       def formatted_total_cost
-        NumberTo.pounds(total_cost)
+        NumberTo.pounds(total_cost(original: true))
```

Comment:
Does it explicitly say somewhere that on the overview page we should show original values rather than adjusted values? I couldn't see it on a skim of the tickets but might have missed it. It just seems a little bit unintuitive UX to me.

Reply:
I queried this with UCD lot and they said it should show the original/requested values when adjustments exist. The prototype does too, although i queried because i was not sure that was intentional

Reply:
Ok nice one, you're on step ahead of me. Then this all looks fine to me and I think David has already approved.